spec/beaker/dsl/roles_spec.rb

```diff
@@ -163,6 +163,14 @@ class ClassMixedWithDSLRoles
       agent1[:type] = 'aio-foss'
       expect( subject.aio_version?(agent1) ).to be === true
     end
+    it 'can take an empty string for pe_ver' do
+      agent1[:pe_ver] = ''
+      expect{ subject.aio_version?(agent1) }.not_to raise_error
+    end
+    it 'can take an empty string for FOSS version' do
+      agent1[:version] = ''
+      expect{ subject.aio_version?(agent1) }.not_to raise_error
+    end
```

Comment:
It seems like a useful set of tests here is to assert what the value of `aio_version?` would be when given a `nil` or `''` value for one or both of `:pe_ver`, and `:version`. I'm guessing that ordering (i.e., we check one before the other) is significant here.  What happens if both `:pe_ver` and `:version` are specified but one is `<` the other (and vice versa)?